packages/manager/modules/pci/src/components/project/empty/empty.html

```diff
@@ -0,0 +1,43 @@
+<div class="container-fluid px-0 pci-project-empty">
+    <div class="row">
+        <div class="col-md-3">
+            <div class="p-5 placeholder">
+                <img data-ng-if="$ctrl.imageSource"
+                    data-ng-src="{{:: $ctrl.imageSource }}"
```

Comment:
Feel free to tell me if you find this more or less readable but why not just using a single `<img>` tag?

```suggestion
                    data-ng-src="{{:: $ctrl.imageSource || $ctrl.placeholder }}"
```

In that way you can get rid of the `ngIf`.